### DISCUSSION TITLE
Words of the day

### DISCUSSION THREAD

**Discussion by O-Hatem** [monkeytypegame/monkeytype#787] [Ideas] [answered]
What do you think of adding a mode where there would be a group of words(may be 10 words) each day for the user to practice on?
Something like the 50 words mode but every day it's a new set of words.
It can go far to be customized for every user depending on the most words the user is mistaken in.

**Comment by Miodec** [comment, marked as answer]
Not really sure whats the point of something like this

**Reply by O-Hatem** [reply]
A good practice to be faster at typing is to memorize the whole word not the set of letters individually.
so i thought if you can practice a set of words every day so you can memorize them without the need to practice in other modes where the words are not guaranteed to show up a number of times you need to memorize them.

**Reply by frogfrogfrogfrogfrog** [reply]
I don't think memorizing words is that important in typing. I guess if your thinking about muscle memory then I guess having a good muscle memory of frequent words is helpful for your typing speed. I think this feature would only apply to people who are just learning how to type but since Monkeytype is a very competitive site (people typing past 190 wpm for an hour straight) a "daily" words feature would kind of be obsolete to those who type at a fast speed like above 100 wpm.

**Comment by SeerLite** [comment]
Maybe there could be a separate leaderboard for this? Have it be something like a "daily challenge". It'd have to be more than 10 words though.
